extension/background/youtubei.js

```diff
@@ -1,8 +1,16 @@
+import { fetchChaptersFromComments } from './background.js'
+
 const INNERTUBE_CLIENT_VERSION = "2.20211129.09.00"
 
 export async function fetchChapters(videoId) {
     const videoResponse = await fetchVideo(videoId)
-    return chaptersFromVideoResponse(videoResponse)
+    let chapters = chaptersFromVideoResponse(videoResponse)
+    if (chapters.length) {
+        return chapters
+    }
+
+    chapters = await fetchChaptersFromComments(videoResponse)
```

Comment:
YouTube API shouldn't know anything about `background.js` let's move this logic there.

Reply:
Do you mean moving the fetchChapters function?

Reply:
Moved the fetchChapters function. Hopefully I've guessed it right.

Reply:
Yes, you are right.